float32 finger
float32 Enabled
float32[3] Pos
float32[3] Torque
float32[3] Velocity
string AllData
